themes/mariana-reference.json

```diff
@@ -153,7 +154,7 @@
 		"statusBar.foreground": "LIGHT_1",
 		"statusBar.background": "DARK_0",
 		"statusBar.debuggingBackground": "ORANGE",
-		"statusBar.debuggingForeground": "DARK_0",
+		"statusBar.debuggingForeground": "DARK_1",
```

Comment:
I could be convinced of changing to `DARK_1`. I chose `DARK_0` for maximum foreground background contrast.

Reply:
Agreed!